dist/listbox/ds4/listbox.css

```diff
@@ -16,6 +16,7 @@ select.listbox__control {
   font-family: "Helvetica Neue", Helvetica, Arial, Roboto, sans-serif;
   font-size: inherit;
   height: 40px;
+  line-height: 1.15;
```

Comment:
I removed the line-height in this PR remember:

https://github.com/eBay/skin/pull/252/files#diff-a919e64bc0f0127334b9fabddcb84118L19

Any reason you are putting it back?

Reply:
Unintended. Will fix.

Reply:
I pulled that file back from `master`, not the 6.0.0. Whoops. ;-)